tests/e2e/steps_double_sign.go

```diff
@@ -52,80 +52,6 @@ func stepsDoubleSignOnProviderAndConsumer(consumerName string) []Step {
 				},
 			},
 		},
-		{
-			// consumer double sign
```

Comment:
Seems this is used to check that the equivocation proposal operates correctly. This is not needed anymore. However, I kept the double singing on the provider that propagates a vote-power change on the consumer because this is still a useful test.